plugins/node/instrumentation-tedious/src/instrumentation.ts

```diff
@@ -78,7 +78,7 @@ export class TediousInstrumentation extends InstrumentationBase {
     return [
       new InstrumentationNodeModuleDefinition(
         TediousInstrumentation.COMPONENT,
-        ['>=1.11.0 <=15'],
+        ['>=1.11.0 <18'],
```

Comment:
nit: It would perhaps be nice to have this version string be the same as is used in the README. However, it is totally fine to leave that for now. 

@blumamir was talking about a separate PR to pick on preferred form (`>=${theMinVer} <=${theMaxMajorVer}`  or >=${theMinVer} <${theMaxMajorVer+1}`) and update many/most of the isntrumentations.

Reply:
I will rebase my PR after this one merges 👍